signer/kms/aws/client.go

```diff
@@ -352,7 +343,14 @@ func (a *awsClient) setupClient(ctx context.Context, ksp *kms.KMSSignerProvider)
 		return fmt.Errorf("loading AWS config: %w", err)
 	}
 
-	a.client = akms.NewFromConfig(cfg)
+	if a.endpoint != "" {
```

Comment:
Looks fine to me I think? I am guessing that there is no longer a way to append `endpoint` to `opts`?

Reply:
Correct. For some reason 🤷 , it can only be done on the service itself (`akms` in this case). Deprecation of `aws.EnpointResolver` is documented here: https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/endpoints/#v1-endpointresolver.